tasks/release.rb

```diff
@@ -61,6 +68,7 @@
 
     task :push => :build do
       sh "gem push #{gem}"
+      sh "npm publish" if File.exist?("#{framework}/package.json")
```

Comment:
Put out a post on the message board, for people to register for npm and send me their usernames